gcp/deployment-packages/autoscale-byol/README.md

```diff
@@ -0,0 +1,130 @@
+# GCP Deployment Manager package for Check Point Autoscaling BYOL solution
+This directory contains CloudGuard IaaS deployment package for Check Point Autoscaling (BYOL) solution published in the [GCP Marketplace](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/checkpoint.vsec?tab=Overview).
```

Comment:
link is to Azure, please change to GCP marketplace link

Reply:
Fixed